src/components/DisplayTime/DisplayTime.js

```diff
@@ -21,14 +21,14 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function DisplayTime() {
+function DisplayTime(props) {
   const classes = useStyles();
 
   return (
     <li className="list-style">
-      <p>01-01-2021 00:00:00
-        <Button variant="contained" size="small" className={classes.margin}>Notify</Button>
-        <Button variant="contained" size="small" className={classes.margin}>Log Feedback</Button>
+      <p>{ props.dispTime }
+        <Button onClick={"hello"} variant="contained" size="small" className={classes.margin}>Notify</Button>
+        <Button onClick={"hello"} variant="contained" size="small" className={classes.margin}>Log Feedback</Button>
```

Comment:
![image](https://user-images.githubusercontent.com/6842987/113639277-b1e4d680-9670-11eb-84d1-25b0e8718f67.png)

The value of onClick property should be a function not a string.

Reply:
You can see these errors in the browser by pressing F12 (or Fn+F12) in the "Console" tab.
![image](https://user-images.githubusercontent.com/6842987/113639381-f2dceb00-9670-11eb-9dd5-e2723b2594da.png)